BoardHandler/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/Opponents.java

```diff
@@ -2,21 +2,35 @@
 
 import pl.jakubowskiprzemyslaw.tajgertim.models.player.Player;
 
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 
 class Opponents {
     private List<Player> opponents;
 
+    Opponents(Player player) {
+        opponents = new ArrayList<>();
+        opponents.add(player);
+    }
+
     Opponents(Player player, Player opponent) {
         opponents = Arrays.asList(player, opponent);
     }
 
+    public void addOpponent(Player player) {
```

Comment:
if there is 2 players in list then maybe some exception throwing ?

Reply:
Will be done later.
Tracked under #140